### PR TITLE
Add dry run mode

### PR DESCRIPTION
**Pull request by jkonecny12**
This will be helpful for development and also for implementation of unit-tests.

### CODE REVIEW

File: scripts/launcher/lib/conf/configuration.py [hunk 1]
```diff
@@ -55,6 +55,22 @@ def __repr__(self):
         return msg
 
 
+class GlobalConfiguration(BaseConfiguration):
+    """Global static configuration usable from all corners of program"""
```
Comment (M4rtinK) [inline scripts/launcher/lib/conf/configuration.py]: I suggest:
"from all parts of the program"

File: scripts/launcher/lib/log_monitor/__init__.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+from lib.utils import is_dry_run, disable_on_dry_run
+from pylorax.monitor import LogMonitor as LoraxLogMonitor
+
+from .log_handler import VirtualLogRequestHandler
+
+
+class LogMonitor(object):
+    def __init__(self, install_log, timeout, log_request_handler_class=None):
+        """Monitor VM logs and reacts properly on what happen there.
```
Comment (M4rtinK) [inline scripts/launcher/lib/log_monitor/__init__.py]: "what happens here"

File: scripts/launcher/lib/conf/configuration.py (hunk 1 above)
Comment (jkonecny12) [reply scripts/launcher/lib/conf/configuration.py]: Fixed.

File: scripts/launcher/lib/log_monitor/__init__.py (hunk 2 above)
Comment (jkonecny12) [reply scripts/launcher/lib/log_monitor/__init__.py]: Fixed

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fixed issues pointed by Martin
* Fixed issues preventing installation (minor changes but big problems)

**Pull request merged**